### PR TITLE
Updated input-output.md

### PR DESCRIPTION
**Pull request by ArshadAriff**
The typos and the errors in the documentation are resolved

### CODE REVIEW

File: docs/docs/scheme-intro/input-output.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ If you first evaluated the code that writes into a file `test-2.scm` you should
 was written into it. Note that `read` was called two times and read two expressions (two strings).
 
 Same as with output-port you need to call `close-input-port` procedure to close the input port. But
-Scheme, same as with output port have handy function that allow to close the port automagically.
+Scheme, same as with output port have handy function that allow to close the port automatically.
```
Comment (jcubic) [inline docs/docs/scheme-intro/input-output.md:162]: This is not an error, see ["automagically" in Jargon File](http://www.catb.org/jargon/html/A/automagically.html).

**Review by jcubic — CHANGES_REQUESTED** [review]

**Comment by ArshadAriff** [comment]
Sir i have renewed the PR

**Comment by jcubic** [comment]
You don't need to clone and open new PR, you can commit here and answer the comments I did in the Code Review. This is not how you contribute. Now I need to do a review again on your new PR.

**Comment by ArshadAriff** [comment]
Oh really sorry sir working on it

**Pull request closed without merging**